squad/core/migrations/0083_auto_20180816_1022.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
this migration needs to be renumbered because #324 which was merged since already has a migration `0083_*`

maybe take the opportunity and also give it a proper name